searx/static/themes/simple/src/less/style.less

```diff
@@ -296,10 +296,26 @@ article[data-vim-selected].category-social {
   padding: 0 5px 25px 0 !important;
 }
 
+.audio-control audio {
+  width: 100%;
+  padding: 10px 0 0 0;
+}
+
+.embedded-content iframe {
+  width: 100%;
+  padding: 10px 0 0 0;
+}
+
 .result-videos .content {
   overflow: hidden;
 }
 
+.result-videos .embedded-video iframe {
+  width: 100%;
+  aspect-ratio: 16 / 9;
```

Comment:
> The property is not supported by one year old browser ..

I think ...

```yaml
@supports not (aspect-ratio: 1 / 1) {
  .result-videos .embedded-video iframe {
    height: calc(@results-width * 9 / 16); // the right value
  }
}
```

is an good addition to support older browsers (thanks for this suggestion).

But I don't want to implement different screen solutions, at least not in this PR.  The intention of this PR is not to get perfect layout we hadn't before, the intention is mostly to modularize HTML in templates.  We can improve the layout afterwards and discuss better layouts, BTW the aspect ratio of videos is not always 16:9 and on small devices it might be better when the "show video" button shows the video at a full screen.

Reply:
FYI I added edc31d84